source/includes/_address.md

```diff
@@ -1,3 +1,158 @@
 # Address
 
 ## Address Object
+
+Attribute | Type | Description
+--------- | ---- | -----
+street_name | `string` | The street name of Address
+building | `string` | The building of Address
+zip_code | `string` | The zip code of Address
+city | `string` | The city of Address 
+country | `string` | The country of Address 
+geo | `GeoPoint` | The geo position of Address
```

Comment:
this geo point doesn't mean anything to the general public, it's a python mongo thing. we need to use global types of the API output, which in this case it's JSON. so I think this is a list with longitude and latitude